scripts/build/set-azp-variables.ps1

```diff
@@ -0,0 +1,8 @@
+# Read variables shared between stages and tasks
+$Dir = "$env:AGENT_BUILDDIRECTORY\azp-variables"
+
+$Version = Get-Conent "${Dir}\SONAR_PROJECT_VERSION"
+Write-Host "Sonar project version is '${Version}'"
+
+# Set the azure build variable (SONAR_PROJECT_VERSION) so it can be used by other tasks
+Write-Host "##vso[task.setvariable variable=SONAR_PROJECT_VERSION;]$Version"
```

Comment:
This weird stuff is taken from the deleted `set-azp-project-verison-variable.ps1`. Git stage didn't handle he rename history nicely.

Docs: https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#set-variables-in-scripts

Reply:
did you use `git mv` for the rename?

Reply:
No, I stage changes before commit and that pairs the deleted/added as rename. There's no much harm for this file.